### DISCUSSION TITLE
I was approved more than 4 days ago and my benefits are still not available

### DISCUSSION THREAD

**Discussion by Tallesco201** [community/community#136858] [GitHub Education]
### Hi everyone,

Is it normal for this to happen? I was approved in github education more than 4 days ago and until now my benefits are not available

![image](https://github.com/user-attachments/assets/c27acaef-2ed7-441e-8c55-307bacf42148)

**Comment by trandangtrungduc** [comment]
According to [this](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved), Github aims to provide you with your educational benefits within 3 days of this approval status. However, during busy periods like the start of a new semester, it may take a bit longer.

**Comment by davevad93** [comment]
Hi @Tallesco201 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Tallesco201 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
